SparkPost/2web2ui
# SA-1216 Add "Per day/minute/hour" for unique metrics based on precision

### What Changed
 - Added a `uniqueLabel` for each item in `metricsRollupPrecisionMap`
 - Added a `isUniquePerTimePeriod` option for metrics
 - If the metric is `isUniquePerTimePeriod`, the Legend will display the current precision's `uniqueLabel` i.e. "Unique clicks per minute"

### How To Test
 - Add metrics to the summary report that have the word Unique in them. ie. Unique confirmed opens
 - Add metrics that are not unique
 - Ensure the Legend displays the correct "Unique Confirmed Opens per hour" based on precision.
 - Ensure the Legend does not display the "per" precision times for non unique metrics
 - Ensure these only are when you have the metrics rollup feature flag turned on.
